packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -93,7 +93,7 @@ registerFluentComponents(fluentProgressRing, fluentDesignSystemProvider);
 
 @customElement('file-list')
 // @customElement('mgt-file-list')
-export class MgtFileList extends MgtTemplatedComponent {
+export class MgtFileList extends BasePersonCardSection {
```

Comment:
Actually I like the interface implementation approach I took in #2057 better this should be changed to reflect that approach